code/modules/mob/living/living_defense.dm

```diff
@@ -374,3 +374,11 @@
 
 /mob/living/proc/cult_self_harm(damage)
 	return FALSE
+
+/mob/living/shove_impact(mob/living/target, mob/living/attacker)
+	if(!IS_HORIZONTAL(src))
+		add_attack_logs(attacker, target, "pushed into [src]", ATKLOG_ALL)
+		target.KnockDown(1 SECONDS) // knock them both down
+		KnockDown(1 SECONDS)
+		return TRUE
```

Comment:
Might be nice to have a specific sound effect here (if there isn't one already) to add some oomph to the action.

Reply:
I hate trying to find sounds. Might see if tg has one